de.metas.business/src/main/java/org/adempiere/impexp/impl/ImportProcessFactory.java

```diff
@@ -35,13 +35,7 @@
 import org.adempiere.impexp.product.ProductImportProcess;
 import org.adempiere.impexp.spi.IAsyncImportProcessBuilder;
 import org.adempiere.model.InterfaceWrapperHelper;
-import org.compiere.model.I_I_BPartner;
-import org.compiere.model.I_I_BPartner_GlobalID;
-import org.compiere.model.I_I_DiscountSchema;
-import org.compiere.model.I_I_Inventory;
-import org.compiere.model.I_I_Product;
-import org.compiere.model.I_I_Request;
-import org.compiere.model.I_I_User;
+import org.compiere.model.*;
```

Comment:
I assume this is something intelliJ does automatically?
Pls coordinate with teo about this; might need to change the IntelliJ settings

Reply:
Fixed imports in bc8787cdead9ee5d906d1e10e49666a36ecbaad7